src/views/app/map.vue

```diff
@@ -1,7 +1,206 @@
 <template>
-  <HelloWorld />
+  <div class = 'container'>
+    <div v-if="currentEvents.length" class='events'>
+      <v-row align="start">
+        <v-col>
+          <h1 style="color: #ff6961" class="my-5 ml-5">Events</h1>
+        </v-col>
+        <v-col align="right" class="my-5 mr-5">
+          <v-btn 
+            rounded="xl"
+            @click="closeClick()"
+            >Close</v-btn
+          >
+        </v-col>
+      </v-row>
+      <v-row align="start">
+        <ListOfItems :items="currentEvents" />
```

Comment:
edit the component so that the 'create event' button doesn't appear, and the photos look good without changing their behavior on the rest of the sites where it is used, or, if you prefer, create a new component.